build.sh

```diff
@@ -1,11 +0,0 @@
-#!/bin/sh -ex
-VERSION=v1.65.2
-git clone https://github.com/rclone/rclone.git
-(
-    cd rclone &&
-    git checkout $VERSION &&
-    make GOTAGS=netcgo
```

Comment:
How do we preserve this? Is it built with this flag by default now?

Reply:
It seems we don't have an alternative for this by default. So if we want this, we should probably build from source. @yocalebo advise please

Reply:
@themylogin why do we need this flag?

Reply:
I'm actually finding it almost impossible to understand what the `netcgo` flags actually do.... I see your commit states it builds rclone so that it uses nscd on the host. However, that flag doesn't seem to exist....at least from what I can find. I see the `cgo` golang module which might be the same....but I have no idea what `netcgo` actually does or if it's actually even doing anything. Also, I'm not sure this was the right decision. rclone is chatty, there is no way around that because it's sending a bunch of http requests to the cloud providers. And each of those requests get resolved.

Reply:
netcgo is about having rclone use nscd. If we bypass nscd there will potentially be thousands of DNS requests while doing rclone jobs (once per op IIRC).

Reply:
Still, though, no one can point me to what netcgo actually is. It doesn't look like it exists.

Reply:
Regardless of how we build it we need to validate it's using nscd via C resolver. If we were doing it wrong initially, then this is good chance to fix it.

Reply:
Regarding the netcgo tag, I found some info here https://go.dev/src/net/conf.go?s=976:994

Now the latest 1.67.0 release, according to the pipeline (https://github.com/rclone/rclone/actions/runs/9518401474/job/26241932157) does not seem to include the netcgo tag

Neithr the make file includes it by default (https://github.com/rclone/rclone/blob/master/Makefile)

So if we want netcgo we have to build it ourselves.

Reply:
@stavros-k thank you very much!

Reply:
I confirm that we need netcgo if we want to use nscd. Perhaps, add a comment this time.